openapi/index.yaml

```diff
@@ -1087,13 +1115,18 @@ definitions:
         minimum: 1
         description: Number of items returned for each page.
         example: 2
+      hasMoreResults:
```

Comment:
We should prefer snake case in API model's definition :)
```suggestion
      has_more_results:
```

Reply:
Ok, this will require a change in fn-commons fillPage.